rest-api-guidelines/guides/api-testing-ci-environment.md

```diff
@@ -31,63 +24,28 @@ The following must be available in the CI environment before testing:
     dredd v2.2.5 (Darwin 16.4.0; x64)
    ```
 
-4. [**Apiary CLI Tool**](https://help.apiary.io/tools/apiary-cli) MUST be installed globally in the CI environment:
-
-   ```text
-    $ gem install apiaryio
-   ```
-
-   ```text
-    $ apiary version
-    0.8.0
-   ```
-
-5. **Apiary API Key** MUST be set in the CI Environment environment variables:
-
-   ```text
-    $ export APIARY_API_KEY=xyz
-   ```
-
-   To obtain an Apiary API key, head to [https://login.apiary.io/tokens](https://login.apiary.io/tokens) \(NOTE: You will need the "ALL" Scope\)
-
 ## Testing an API
 
 ### Test Run Prerequisites
 
 To test an API within the CI environment provisioned as mentioned in the environment prerequisites, you will need the following:
 
-1. The name \(subdomain\) of API project at Apiary
+1. A `swagger.yaml` file with the description of API being tested
 
-   ```text
-    $ export APIARY_API_NAME=bomapi3
-   ```
+   The file should be fetched from [API Design Platform](design-plaform.md)
 
-   > See [How to find the Apiary API name](https://help.apiary.io/faq/find-api-name/) for more details.
```

Comment:
We need to update with the equivalent that we have in swaggerhub.

Reply:
The system names were migrated as such from apiary and they are the names of the APIs in swaggerhub (the display names). They are going to introduce some changes later to the product's UI to be able to choose which display name will be shown, so it may change in the future.

Reply:
Ok, so the new version uses values of _**title**_ from OpenAPI spec as the API title. The search works on those titles **AND** the system names. Here is the link to the online doc on search: https://app.swaggerhub.com/help/ui/searching